Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG GO_VERSION=1.20.1
+ARG VARIANT=bullseye
+FROM golang:${GO_VERSION}-${VARIANT} as builder
```

Comment:
This `builder` stage is currently unused, and I want to change the way xk6 is currently built from `go install` to copying the checked out repo in the container and running `go build` instead. This would make it work with forks, which is currently not possible with `go install`.